src/collection.ts

```diff
@@ -1151,6 +1200,34 @@ export class Collection<
     )
   }
 
+  /**
+   * Retrieves one document from the KV store according to the given options.
+   *
+   * If no options are given, the first document in the collection is retreived.
+   * 
+   * @example
+   * ```ts
+   * // Get the first user
+   * const { result } = await db.users.getOne()
+   *
+   * // Get the first user with username that starts with "a"
+   * const { result } = await db.users.getOne({
+   *   filter: doc => doc.value.username.startsWith("a")
+   * })
+   * ```
+   *
+   * @param options - List options, optional.
+   * @returns A promise that resovles to the retreived document
+   */
+  async getOne(options?: ListOptions<Document<TOutput>>) {
+    // Get and return one document
+    return await this.handleMany(
```

Comment:
I think users using a method named `getOne()` might expect the return type to be just a document value (or null if not found), also making it easier to use as you would not need to destructure the `result` value and extract the first item in the list. So alternatively this could be:
```ts
const { result } = await this.handleMany(
  this._keys.id,
  (doc) => doc,
  { ...options, resultLimit: 1 }
)

return result.at(0) ?? null
```